CWP/papers/roadmap/authors/address.txt

```diff
@@ -0,0 +1,112 @@
+ANL: High Energy Physics Division, Argonne National Laboratory, Argonne IL, USA
+BNL: Physics Department, Brookhaven National Laboratory, Upton NY, USA
+Caltech: California Institute of Technology, Pasadena, California, USA
+CBPF: Centro Brasileiro de Pesquisas Fisicas, Rio de Janeiro, Brazil
```

Comment:
Should be F\\'{i}sicas (acute accent, or use the unicode).

Reply:
All these mistakes are coming from the copy/paste from the PDF (experiment articles) that tended to mess all the accented characters...